src/Tizen.NUI.Wearable/src/public/CircularPagination.cs

```diff
@@ -487,6 +487,10 @@ public Position GetIndicatorPosition(int index)
         [EditorBrowsable(EditorBrowsableState.Never)]
         public virtual void SetIndicatorPosition(int index, Position position)
         {
+            if (position == null)
```

Comment:
could you add \<exception cref= ...\> here?
(example) https://github.com/Samsung/TizenFX/blob/d8c72a9533502bd3382ec1e1db12e57614c2a710/src/Tizen.NUI/src/public/Window.cs#L1414

Reply:
done